libs/build_pio.sh

```diff
@@ -32,6 +32,7 @@ if $MODULES; then
     fi
 else
     prefix=${PIO_ROOT:-"/usr/local"}
+    PNETCDF=${PNETCDF:-"/usr/local"}
```

Comment:
This is not correct.

Reply:
I encountered an issue when building the stack with pnetcdf enabled.  I had PNETCDF defined outside of this script, but when I ran the script, it blew up complaining about an undefined PNETCDF.  Maybe it was user error.  I'll just remove it.